docs/source/release_notes.rst

```diff
@@ -13,7 +13,8 @@ Future Release
     * Testing Changes
        * Add GitHub Actions cache to speed up workflows (:pr:`2475`)
        * Fix latest dependency checker install command (:pr:`2476`)
-       
+       * Add pull request check to CI workflow (:pr:`2477`)
```

Comment:
nit: Should we clarify and say `Add pull request check for linked issues to CI workflow` to be more specific about the check?